aggregator/computers.go

```diff
@@ -0,0 +1,16 @@
+package aggregator
+
+import "sort"
+
+func computeMedian(nums []float64) float64 {
+	sort.Float64s(nums)
+
+	numsLen := len(nums)
+	mid := numsLen / 2
+
+	if numsLen&1 != 0 {
+		return nums[mid]
+	}
+
+	return (nums[mid-1] + nums[mid]) / 2
```

Comment:
shouldn't this method do the sum of all of them? this is not accurate for non consecutive values.
eg.: 1, 1.01, 1.03, 1.05, this method returns 1.02 when in fact the median of the values is 1.0.225
same thing applies if the last one is 0.03 and so on

Reply:
as discussed, we should keep the median approach as it eliminates the extreme values